### ISSUE TITLE
Four decimal places on Invoices and other items

### ISSUE THREAD

**Issue by gkavaliu** [XeroAPI/xero-php-oauth2#17]
Hello,

I wanted to be able to use Four decimal places. Currently this is not possible as far as I can tell.

I modified the code and was able to make it work.

Below are some of the changes. 

Note the /Invoices#bulk resourcePath does not seem to accept the query param unitdp

```
//$resourcePath = '/Invoices#bulk';
$resourcePath = '/Invoices';

protected function createInvoicesRequest($xero_tenant_id, $invoices, $summarize_errors = true, $unitdp = null)

// query params
if ($unitdp !== null) {
	$queryParams['unitdp'] = AccountingObjectSerializer::toQueryValue($unitdp);
}
```
Thank you,
Glen Kavaliunas

**Comment by toby2g** [comment]
Wanted to add on that this is occurring for other items. i.e. Contacts

The query parameter is being concatenated onto the end of the resourcePath. Which, when used in the request, is considered to be a part of the fragment, not an actual query parameter.

**Comment by SidneyAllen** [comment]
Hi @toby2g 

This is fixed in version 1.0.6.  Give it a whirl!
